pyproject.toml

```
@@ -99,6 +98,9 @@ dependencies = [
   [tool.setuptools_scm]
   write_to = "src/insight/version.py"
 
+  [tool.setuptools.package-data]
+  synthesized = ["fonts/inter-v3-latin-regular.ttf", ]
```

This should be adding the font to the insight package. Currently, I don't think it does.
```suggestion
  insight = ["src/insight/fonts/inter-v3-latin-regular.ttf"]
```